CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Note that releases after 0.4.0 will aim to preserve the transaction wire format.
 * *chain-abci* [1292](https://github.com/crypto-com/chain/pull/1292): implements new punishment and staking state transition specification and new nonce logic
 * *chain-storage* [1424](https://github.com/crypto-com/chain/pull/1424): add two columnes to the database
 * *chain-storage* [1428](https://github.com/crypto-com/chain/pull/1428): change merkle tree of staking states to jellyfish-merkle
+* *chain-storage* [1466](https://github.com/crypto-com/chain/pull/1466): add a colume to store historical staking versions
```

Comment:
not a part of 0.4.0 -- 0.5.0?

Reply:
but this PR isn't breaking -- 0.4.1? As "account" path is preserved and this JSON-encoded version is on a new path?

Reply:
oh, it's breaking because of the new column

Reply:
Moved to 0.5.0.